Back-End/VaraticPrim/VaraticPrim/Controllers/CounterController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AutoMapper;
+using FluentValidation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using VaraticPrim.Framework.Exceptions;
+using VaraticPrim.Framework.Managers;
+using VaraticPrim.Framework.Models.CounterModels;
+using VaraticPrim.Framework.Models.LocationModels;
+using VaraticPrim.Framework.Models.UserModels;
+
+namespace VaraticPrim.Controllers;
+
+[Route("[controller]")]
+public class CounterController : ApiBaseController
+{
+    private readonly CounterManager _counterManager;
+
+    public CounterController(CounterManager counterManager)
+    {
+        _counterManager = counterManager;
+    }
+    
+    [HttpPost]
+    public async Task<IActionResult> Create([FromBody] CounterCreateModel counter)
+    {
+        try
+        {
+            return Ok(await _counterManager.Create(counter));
+        }
+        catch (ValidationException e)
+        {
+            return ValidationError(e);
+        }
+    }
+    
+    [HttpGet("{id:int}")]
+    public async Task<IActionResult> Get([FromRoute] int id)
+    {
+        try
+        {
+            return Ok(await _counterManager.GetById(id));
+        }
+        catch (CounterNotFoundException)
+        {
+            return BadRequest("counter_not_found", "Counter not found");
```

Comment:
create a `ApiErrorCodes` class where will be stored all this errors and same for the messages